spec/src/test/java/impl/ThreadMessageHandlerTest.java

```diff
@@ -94,13 +92,20 @@ public void shouldCreateNewThreadWhenNewThreadIsCreatedInSlack() {
         when(slackResourceMock.getUserEmail("user_id")).thenReturn(just(user.getEmail()));
         when(slackResourceMock.getUserEmail("original_message_user")).thenReturn(just(originalMessageUser.getEmail()));
         when(slackResourceMock.getMessageFromSlack(eq("channel"), eq(message.getTs()))).thenReturn(just(message));
-
+        when(slackResourceMock.getAuth(any()))
+            .then(invocation -> {
```

Comment:
I've never seen the when/then construction, is this an alias for thenAnswer?

Reply:
yup